src/Microsoft.Data/ArrowStringColumn.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.Data
     /// <summary>
     /// An immutable to hold Arrow style strings
     /// </summary>
-    public partial class ArrowStringColumn : BaseColumn, IEnumerable<string>
+    public partial class ArrowStringColumn : DataFrameColumn, IEnumerable<string>
```

Comment:
All other classes now have the `DataFrameColumn` suffix, but this one doesn't. Is there a reason for that?

Reply:
Renamed to `ArrowStringDataFrameColumn` now